ursula_cli/shell.py

```diff
@@ -106,6 +106,10 @@ def run(args, extra_args):
     if os.path.isfile(ansible_var_defaults_file):
         _append_envvar("ANSIBLE_VAR_DEFAULTS_FILE", ansible_var_defaults_file)
 
+    ansible_ssh_config_file = "%s/ssh_config" % args.environment
```

Comment:
As this is a general-purpose tool I would like this to be a bit smarter about whether or not this is set.  We happen to use ssh config files, but I'm guessing many do not.  So, I would at least check to see that the ssh_config file exists before including it.  

Also, instead of string interpolation, we might want to use os.path.join.